src/SFA.DAS.ProviderApprenticeshipsService.Web/DependencyResolution/DefaultRegistry.cs

```diff
@@ -216,6 +218,14 @@ private ProviderApprenticeshipsServiceConfiguration GetConfiguration(string envi
             return configurationService.Get<ProviderApprenticeshipsServiceConfiguration>();
         }
 
+        private EncodingConfig GetEncodingConfig(string environment, IConfigurationRepository configurationRepository)
```

Comment:
Is it worth moving this into a separate class? It could then be resolved in the configuration expression e.g. `For<EncodingConfig>().Use(c => c.GetInstance<IConfigurationService>().Get<EncodingConfig>("SFA.DAS.Encoding", environment));`